view/frontend/templates/topscript.phtml

```diff
@@ -0,0 +1,394 @@
+
```

Comment:
Wouldn't it make sense to split the file into two files - `autocomplete-script.phtml` and `instantsearch-script.phtml` ?